p2p/transport/webrtc/connection.go

```diff
@@ -166,6 +173,13 @@ func (c *connection) OpenStream(ctx context.Context) (network.MuxedStream, error
 	if id > math.MaxUint16 {
 		return nil, errors.New("exhausted stream ID space")
 	}
+	// Limit the number of streams, since we're not able to actually properly close them.
+	// See https://github.com/libp2p/specs/issues/575 for details.
+	if id > maxDataChannelID {
+		c.Close()
+		return c.OpenStream(ctx)
```

Comment:
here too?

Reply:
That's already a return, no?

Reply:
okay. It will return the correct error from the call.